src/RPackage-Core/RPackage.class.st

```diff
@@ -514,14 +500,14 @@ RPackage >> importProtocol: aProtocol forClass: aClass [
 RPackage >> includesClass: aClass [
 	"Returns true if the receiver includes aClass in the classes that are defined within it: only class definition are considered - not class extensions"
 
-	^ classes includes: aClass instanceSide
+	^ self definedClasses includes: aClass instanceSide
 ]
 
 { #category : #testing }
 RPackage >> includesClassNamed: aSymbol [
 	"Returns true if the receiver includes class named aSymbol in the classes that are defined within it: only class definition are considered - not class extensions"
 
-	^ classes anySatisfy: [ :class | class name = aSymbol ]
+	^ self definedClassNames includes: aSymbol
```

Comment:
This now requires making 2 intermediate collections. This could be reduced to 1 by only replacing `classes` by `self definedClasses` and keeping the previous `anySatisfy:`.